src/list/actions.js

```diff
@@ -345,9 +345,10 @@ export function selectTabMonitor() {
   };
 }
 
-export function openFAQ() {
+export function openFAQ(close) {
```

Comment:
this can be ditched, but thought it better aligned with user expectations (closing the doorhanger when clicking on links therein).